meta-mender-core/recipes-bsp/u-boot/patches/0002-Generic-boot-code-for-Mender.patch

```diff
@@ -168,6 +168,10 @@ index 0000000..705c39e
 +                                                                        \
 +    "mender_setup="                                                     \
 +    "setenv mender_kernel_root " MENDER_STORAGE_DEVICE_BASE "${mender_boot_part}; "    \
++    "if test ${mmcargs} != \"\"; " \
++    "then "                        \
++    "run mmcargs; "                \
++    "fi; "                         \
```

Comment:
IMHO, we should keep this out of our boot script unless `mmcargs` is really ubiquitous, and encourage calling it outside, before getting into mmcargs. I think we should be careful about this to our central boot script to avoid accumulating baggage there that we will later have to support.

Is this solely because the auto-patcher doesn't respect it and hence it doesn't get called?